app/jobs/update_tokens_job.rb

```diff
@@ -1,9 +1,13 @@
+# frozen_string_literal: true
+
 class UpdateTokensJob < ApplicationJob
+
   queue_as :default
 
   def perform
-    users = User.all 
+    users = User.all
     users.each { |user| AccessTokenRefresher.new.call(user) }
```

Comment:
```suggestion
    User.all.each { |user| AccessTokenRefresher.new.call(user) }
```
For training purposes check out some methods to process users in batches. Maybe your app will have 1231242141 users soon :D